programs/vote/src/vote_instruction.rs

```diff
@@ -413,7 +413,21 @@ pub fn process_instruction(
             } else {
                 None
             };
-            vote_state::withdraw(me, lamports, to, &signers, rent_sysvar.as_deref())
+            let clock_if_feature_active = if invoke_context.is_feature_active(
```

Comment:
Aha. I was wondering why you had to redo this change, but I see the refactor to `programs/vote/src/vote_processor.rs` doesn't exist on v1.8. Can you also remove that dangling file here?